web/src/components/pages/MyIntegrations/index.tsx

```diff
@@ -1,15 +1,18 @@
 import { AuthenticationRequiredPage } from "@reearth-cms/auth";
+import WorkspaceMenu from "@reearth-cms/components/molecules/Common/WorkspaceMenu";
 import Dashboard from "@reearth-cms/components/organisms/Dashboard";
 import MyIntegrations from "@reearth-cms/components/organisms/Settings/MyIntegrations";
 
-const MyIntegrationsPage: React.FC = () => {
+const WorkspacePage: React.FC = () => {
```

Comment:
Why are you renaming this WorkspacePage when it is the MyIntegrations page? Are they the same now?

Reply:
Fixed.
Typos, copied and pasted the name of the component by accident.